packages/nextra/src/compile.ts

```diff
@@ -103,13 +107,19 @@ export async function compileMdx(
   const format =
     _format === 'detect' ? (filePath.endsWith('.mdx') ? 'mdx' : 'md') : _format
 
+  // https://github.com/shuding/nextra/issues/1303
+  const isFileOutsideCWD =
+    !isPageImport && path.relative(CWD, filePath).startsWith('../')
```

Comment:
```suggestion
    !isPageImport && path.relative(CWD, filePath).startsWith('..')
```

Probably breaks windows